### DISCUSSION TITLE
Brunch installed on HP Chromebook 14 4G but fails to login into Google/no Wifi Connection

### DISCUSSION THREAD

**Discussion by Nyta8591** [sebanc/brunch#1845] [Q&A]
I have this issue with my HP Chromebook 14 4G [tried](https://chromebookdb.com/chromebook/hp-chromebook-14-14-ak040nr/) Kernel 15.15 and 15.10, but it keeps getting stuck at Google Login page and no wifi in Guest mode (doesn't load anything on Chrome even though says Wifi connected sucessfully). I also tried loading the latest r111 recovery [https://github.com/sebanc/brunch/releases/tag/r111-stable-20230414](https://github.com/sebanc/brunch/issues/url) through sudo chromeos-update -r/recovery.bin by mounting USB drive, it was installed sucessfully, but didn't help.....

Also the audio and bluetooth won't work on this device, even though though I tried activating "chromebook_audio" via Framework options. I managed to extract Audio drivers from original ChromeOS, can I mount them somehow into Brunch???

Here are my lsmod outputs from original ChromeOS. Can someone help plssss? Thanks so much!!!

[lspci, lsusb, lsmod.txt](https://github.com/sebanc/brunch/files/11636532/lspci.lsusb.lsmod.txt)

**Comment by Ares-Hantrio** [comment]
Can you share your system information in detail..
